e2e/tests/smoke_test.js

```diff
@@ -0,0 +1,16 @@
+/* global process */
+
+const user = {
+  password: 'Password12',
+  email: 'solicitor@example.com'
+}
```

Comment:
I'd put it in separate config file already

Reply:
makes sense